files/tests/test_secure_media_thumbnails.py

```
@@ -0,0 +1,303 @@
+    NOTE: These tests require database access. Run with:
+        make docker-up  # Start PostgreSQL
+        python manage.py test files.tests.test_secure_media_thumbnails.SecureMediaViewGetMediaFromPathTests
+    """
+    databases = ['default']
+
+    def setUp(self):
+        self.view = SecureMediaView()
+
+    def test_get_media_from_thumbnail_path_returns_none_when_no_media(self):
+        """When no media matches the thumbnail path, return None."""
+        media, actual_path = self.view._get_media_from_path(
```

Ruff (0.14.13)

Unpacked variable `actual_path` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)